### PR TITLE
[#770] Export User Info W Deposits

### LINKED ISSUE #770: Export User Info
Output to csv file.

### PR DESCRIPTION
**Pull request by sharpattack**
Export script updated to include total deposits and last deposit date.
closes #770

### CODE REVIEW

**Comment by Meowcenary** [comment]
I'm going to wrap up the header lookup and then check this out. I think I have something "good enough" for now.

File: lib/scripts/users_list.rb [hunk 1]
```diff
@@ -26,4 +29,16 @@ def select_all_users
     User.all.select(:id, :formal_name, :username, :email,
                     :current_sign_in_at, :last_sign_in_at)
   end
+
+  def solr_query_by_depositor(depositor)
```
Comment (Meowcenary) [inline lib/scripts/users_list.rb:33]: I like these tiny methods. They make it much easier to understand what's going on here.

File: lib/scripts/users_list.rb [hunk 2]
```diff
@@ -26,4 +29,16 @@ def select_all_users
     User.all.select(:id, :formal_name, :username, :email,
```
Comment (Meowcenary) [inline lib/scripts/users_list.rb:29]: Why is this at the end? Is it for testing to verify the output?

**Review by Meowcenary — APPROVED** [review]
Looks good to me. Had some small questions, but nothing that should stop you from merging.

File: lib/scripts/users_list.rb (hunk 2 above)
Comment (sharpattack) [reply lib/scripts/users_list.rb:29]: Do you mean the method itself? It was here for the same reason from your other comment: tiny methods to make it easier to understand. Or do you mean the `select` here? Or something else?

Comment (Meowcenary) [reply lib/scripts/users_list.rb:29]: I mean't why is it running `select` for all the users at the end? What is using the returned selection?

Comment (sharpattack) [reply lib/scripts/users_list.rb:29]: Ah, I see. `select` here just retrieves the certain fields I need, omitting all the other fields. https://apidock.com/rails/v4.2.7/ActiveRecord/QueryMethods/select

**Pull request merged**